### ISSUE TITLE
USBHelperLauncher crashes at launch (before GUI)

### ISSUE THREAD

**Issue by Lucide** [FailedShack/USBHelperLauncher#44]
On my machine it crashes before showing any UI, it opens a terminal and then closes.
I tried running the launcher with `--verbose` and/or `--nopatch` but with no "verbose" log file is generated. It does generate configuration files like `conf.json` tho. I've tested the program on other machines and it works fine, even if using the same copied-pasted folder. The error showed in Windows' event viewer is:
```
Application: USBHelperLauncher.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.DirectoryNotFoundException
   at System.IO.__Error.WinIOError(Int32, System.String)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].CommonInit()
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]..ctor(System.String, System.String, System.String, System.IO.SearchOption, System.IO.SearchResultHandler`1<System.__Canon>, Boolean)
   at System.IO.Directory.GetFiles(System.String)
   at USBHelperLauncher.Program.Main(System.String[])
```
I really don't know where I could find additional logs/info, so this is all I got for now.

OS: Windows 10 Pro Version 10.0.18362 Build 18362, I also tried to run the program from different locations and privileges to verify that permissions issues weren't the cause

**Comment by simonj0** [comment]
Windows 10 Build 18362 is very recent, I'm relatively sure we haven't tested the program on that version yet.
It looks like it fails to access the key container and tries to delete it, but some parent directory cannot be found, Could you check if the directory `%APPDATA%\Microsoft\Crypto\RSA\<some SID>` exists?

**Comment by Lucide** [comment]
Oh, ok, I can see why it gets upset then.
I've set up that pc to have all the user's folders like "Documents", "Desktop", ..., and "**Roaming**" on the hdd instead of the ssd.
I did that "the windows way", by setting the location in the "path" field under proprieties;
that works very well, but some files are kept in the original location, probably for compatibility.
![Cattura](https://user-images.githubusercontent.com/33134209/58814842-c1370b00-8626-11e9-82db-55b5ac00b033.PNG)
So, `\Microsoft\Crypto\RSA\` is actually still in `C:\Users\Nemo\User\Roaming` even if `%appdata%` points, in this case, to `D:\Users\Nemo\Roaming`. For now I quick fix i could do is to create a Junction that redirects to C:/..../. I'll try that and post results.
